Infrastructure-Support-of-Publication-Process.md

```diff
@@ -0,0 +1,35 @@
+# Infrastructure Support for DevRel Publication Process
+
+This process provides instructions for setting up and updating the [EOS Dev Portal](docs.eosnetwork.com). 
```

Comment:
You must use a fully qualified domain for external links, such as `https://docs.eosnetwork.com` instead of `docs.eosnetwork.com` or the link will think it's internal

Also, Docs are not the "EOS Dev Portal", they are "EOS Documentation"

Reply:
Done

Reply:
Done